src/components/MedicationGroup.spec.js

```diff
@@ -11,16 +12,46 @@ describe('MedicationGroup', () => {
           { id: 2, medName: 'Enalapril' },
           { id: 3, medName: 'Prednisolon' },
         ]}
+        editMode={false}
+        handleDeleteClick={jest.fn()}
+        handleEditClick={jest.fn()}
       />
     )
 
     const time = screen.getByRole('time')
     const medList = screen.getByRole('list')
     const meds = screen.getAllByRole('listitem')
-    const button = screen.getByRole('button', { name: '-' })
     expect(time).toBeInTheDocument()
     expect(medList).toBeInTheDocument()
     expect(meds).toHaveLength(3)
-    expect(button).toBeInTheDocument()
+  })
+
+  it('renders edit and delte button in edit mode', () => {
+    const handleDelete = jest.fn()
+    const handleEdit = jest.fn()
+
+    render(
+      <MedicationGroup
+        id={'1234'}
+        time={'8:00'}
```

Comment:
Time as a string makes me wonder wether you could go with a time format here

Reply:
I thought about it and in the end I thought it wasn't worth the effort. I guess for now I will keep it simple as it is and as soon I need I will have a look into it. A use case could be that I want to send an alarm when the medication group is due but this needs a work around for IOS so maybe later 😄